cool_seq_tool/data_sources/mane_transcript_mappings.py

```diff
@@ -4,7 +4,8 @@
 
 import pandas as pd
 
-from cool_seq_tool import MANE_SUMMARY_PATH, logger
+from cool_seq_tool import logger
```

Comment:
Did you want to import the logger here? `cool_seq_tool/routers/default.py` and `cool_seq_tool/routers/default.py` also import logger